src/components/InnsendingExcelFil.ts

```diff
@@ -0,0 +1,50 @@
+import env from "../util/environment";
+import {tabellFeil} from "./feilvisning/FeilTabell";
+
+export default (file: File): Promise<tabellFeil[]> => {
+
+  const formData = (file) => {
+    let formData = new FormData();
+    if (file) {
+      // @ts-ignore
+      formData.append(file.name, file)
+    }
+    return formData
+  }
+
+  return fetch(env.baseUrl + '/api/v1/bulk/upload', {
+    method: 'POST',
+    body: formData(file),
+  }).then((response) => {
+    switch (response.status) {
+      case 401: {
+        window.location.href = env.loginServiceUrl;
```

Comment:
Testene dekker ikke dette scenarioet.

Reply:
Nei, vi har ikke lagt til håndteringen for at token er gått ut på Excel enda, så jeg tenkte å vente med den testen til vi har riktig logikk